CMakeLists.txt

```diff
@@ -243,31 +265,41 @@ endif()
 option(USE_OPTIMIZED_BLAS "Whether or not to use an optimized BLAS library instead of included netlib BLAS" OFF)
 
 # Check the usage of the user provided BLAS libraries
-if(BLAS_LIBRARIES)
-  include(CheckFortranFunctionExists)
-  set(CMAKE_REQUIRED_LIBRARIES ${BLAS_LIBRARIES})
-  CHECK_FORTRAN_FUNCTION_EXISTS("dgemm" BLAS_FOUND)
-  unset(CMAKE_REQUIRED_LIBRARIES)
-  if(BLAS_FOUND)
-    message(STATUS "--> BLAS supplied by user is WORKING, will use ${BLAS_LIBRARIES}.")
+if(USE_OPTIMIZED_BLAS)
+  if(BLAS_LIBRARIES)
+    include(CheckFortranFunctionExists)
+    set(CMAKE_REQUIRED_LIBRARIES ${BLAS_LIBRARIES})
```

Comment:
please append. set overwrites user supplied values.

Reply:
So
```
append(CMAKE_REQUIRED_LIBRARIES ${BLAS_LIBRARIES})
```
?

Reply:
OK, thanks. I'll try to look into it next time I free a time slot in my brain for this mess. At the moment I toy with the idea of getting out of all that packaging business. I could actually write programs that do something … maybe that impulse will pass …